CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# [2.2.0]
```

Comment:
```suggestion
# [Next]
```

Reply:
2.2.0 has not been published yet. The changelog entry might be misleading. Should we mark it as next instead? Does this conflict with the `next` branch?